Tests/Application/IOS/ViewController.m

```diff
@@ -0,0 +1,41 @@
+//
+//  ViewController.m
+//  ECLoggingIOSTest
+//
+//  Created by Sam Deane on 26/07/2017.
+//  Copyright © 2017 Elegant Chaos. All rights reserved.
+//
+
+#import "ViewController.h"
+
+@interface ViewController ()
+
+@end
+
+#pragma mark - Channels
+
+ECDefineDebugChannel(LoggingSampleViewControllerChannel);
+
+@implementation ViewController
+
+
+#pragma mark - View lifecycle
+
+- (void)viewDidAppear:(BOOL)animated
+	{
```

Comment:
Historical - ECLogging is my library, not Bohemian's, and it predates my work here so it's a bit of a mix.